src/providers/ollama/api.ts

```diff
@@ -0,0 +1,28 @@
+import { ProviderAPIConfig } from "../types";
+
+const OllamaAPIConfig: ProviderAPIConfig = {
+  headers: () => {
+    return null;
+  },
+  chatComplete: "/v1/chat/completions",
+  embed:"/api/embeddings",
+  getEndpoint: (fn: string, API_VERSION: string, url?: string) => {
+    let mappedFn = fn;
+    if (fn === "proxy" && url && url?.indexOf("/chat/completions") > -1) {
```

Comment:
This is not required as /chat/completions will always point to the unified route.